docs/datastorage/config.md

```diff
@@ -37,12 +46,10 @@ public class ExampleMod {
   public ExampleMod() {
     //...
     
-    // You can also register client only config or server only config
+    // Register the config here
+    // You can also register client only config or server only config by specifying ModConfig.Type.CLIENT or ModConfig.Type.SERVER
```

Comment:
It’s not exactly server only as it is synced to the client. (IMO a better name for Server is actually Common as common is pretty much the same as client but that’s an issue for Forge not here)